### PR TITLE
Update to new JS version

### PR DESCRIPTION
**Pull request by alberthsies**
btw discover one issue, since all the login data are in the original `JS`.
The new `JS` isn't able to use the old firebase data to login

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -22,26 +37,19 @@
...
     print('Bot mode: ' + mode)
     starting_time = str(datetime.now())
-    
-    driver = webdriver.Chrome('./chromedrivers/chromedriver')
-    driver.get(config.config['url'])
-
-    try:
-        #dismiss the cookie message since it make the close button untouchable
-        driver.find_element_by_xpath('/html/body/div[1]/div/a').click()
-        print('dismissed cookie')
```
Comment (whtsead213) [inline bot.py]: i think this can not be removed otherwise selenium may not be able to click some button

Comment (whtsead213) [reply bot.py]: i mean you need to click the dismiss cookie message

File: bot.py [hunk 2]
```diff
@@ -22,26 +37,19 @@
...
-        pass
-
 
     if mode == 'random' or mode == 'r':
+        driver = create_driver(url="url")
         for i in range(int(sys.argv[2])):
             try:
-                random.choice(scenario_list)(driver)
+                scenario = random.choice(scenario_list)
+                firebaseDAO.normal_record(str(scenario), str(datetime.now()), "PLEASE TYPE YOUR NAME HERE")
```
Comment (whtsead213) [inline bot.py]: can you move the input to config.py so we can change the input easier

File: bot.py [hunk 3]
```diff
@@ -72,7 +75,18 @@
...
+                        attack_type = "xss"
+                    elif index == 2:
+                        attack_url = "sql_url"
+                        attack_type = "sql_inj"
+                    else:
+                        attack_url = "normal_url"
+                        attack_type = "other"
+                    driver = create_driver(url=attack_url)
                     try:
+                        firebaseDAO.attack_record(attack_type=attack_type, attack_scenario=str(attack_scenario_list[index]), attack_time=str(datetime.now()), attacker="PLEASE TYPE YOUR NAME HERE")
```
Comment (whtsead213) [inline bot.py]: same, move to config.py

File: bot.py (hunk 1 above)
Comment (alberthsies) [reply bot.py]: I didn't remove it. It is placed into the `create_driver` function

File: bot.py (hunk 2 above)
Comment (alberthsies) [reply bot.py]: Sure NP!

File: bot.py (hunk 3 above)
Comment (alberthsies) [reply bot.py]: solved

**Pull request merged**